### PR TITLE
Eran/remove etcd patch

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: src/installer/installer.go [hunk 1]
```diff
@@ -240,14 +240,7 @@ func (i *installer) waitForControlPlane(ctx context.Context, kc k8s_client.K8SCl
 		i.log.Errorf("Timeout waiting for master nodes, %s", err)
 		return err
 	}
-
-	if err := kc.PatchEtcd(); err != nil {
```
Comment (romfreiman) [inline src/installer/installer.go:244]: u're missing the unpatch

Comment (eranco74) [reply src/installer/installer.go:244]: Removed the uppatch from the controller and published a new image
quay.io/eranco74/assisted-installer:test

**Comment by carbonin** [comment]
I think you can also remove the related methods from `src/k8s_client/mock_k8s_client.go` and `src/k8s_client/k8s_client.go`

**Comment by YuviGold** [comment]
Resolved on https://github.com/openshift/assisted-installer/pull/115
/close

**Pull request closed without merging**